WordPress/src/main/java/org/wordpress/android/ui/activitylog/BackupDownloadProgressChecker.kt

```diff
@@ -0,0 +1,61 @@
+package org.wordpress.android.ui.activitylog
+
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.isActive
+import kotlinx.coroutines.withContext
+import org.wordpress.android.fluxc.action.ActivityLogAction.FETCH_BACKUP_DOWNLOAD_STATE
+import org.wordpress.android.fluxc.model.SiteModel
+import org.wordpress.android.fluxc.store.ActivityLogStore
+import org.wordpress.android.fluxc.store.ActivityLogStore.FetchBackupDownloadStatePayload
+import org.wordpress.android.fluxc.store.ActivityLogStore.OnBackupDownloadStatusFetched
+import org.wordpress.android.modules.DEFAULT_SCOPE
+import javax.inject.Inject
+import javax.inject.Named
+import javax.inject.Singleton
+
+@Singleton
+class BackupDownloadProgressChecker
+@Inject constructor(
```

Comment:
Nitpick: I'd consider removing this line wrap. Unfortunately not all, but most classes in the project put the `constructor` keyword on the same line as the class name. The RewindProgressChecker is not consistent so we might want to update that in this PR too since it's a very minor change. Wdyt?

Reply:
Will do 👍